roles/k8s-ut/tasks/main.yml

```diff
@@ -15,7 +30,12 @@
       set -o errexit
 
       echo 'This is a script to run k8s UT tests'
-      useradd nonroot
+
+      if [[ $(uname -m) == "s390x" ]]; then
+        useradd -s /bin/bash -d /home/nonroot -m nonroot
+      else
+        useradd nonroot
+      fi
```

Comment:
can we do same instructions on both the platform?
cc @Rajalakshmi-Girish

Reply:
Tested `useradd -s /bin/bash -d /home/nonroot -m nonroot` on centos:stream9. It seems to work fine. Hence, removed the arch based check.